pkg/controller/certificates/util.go

```diff
@@ -37,6 +37,10 @@ import (
 	"github.com/cert-manager/cert-manager/pkg/util/pki"
 )
 
+const (
+	PartOfCertManagerLabelValue = "certificates"
+)
```

Comment:
nitpick (non-blocking): I don't really have a great concise suggestion here, but I read "PartOfCertManagerLabelValue" to mean "the label value which indicates that the resource is part of cert-manager"

I think this is actually more like "the label value which indicates that the resource is a part of a cert-manager certificate".

Could the variable name be more descriptive of this? Am I reading it wrong? Added a suggestion, but I don't think this is a blocker in any case!

```suggestion
const (
	PartOfCertManagerCertificateLabelValue = "certificates"
)
```

Reply:
yes, I think your suggestion reads better and matches what this is better as it _is_ a 'certificate' specific value for the 'part-of' key  👍🏼